.ci/oci-nightly-olm.sh

```diff
@@ -17,46 +17,71 @@
 set -e
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
-# error on unset variables
-set -u
 
 export OPERATOR_REPO=$(dirname $(dirname $(readlink -f "$0")));
 source "${OPERATOR_REPO}"/.github/bin/common.sh
 source "${OPERATOR_REPO}"/.github/bin/oauth-provision.sh
 
-#Stop execution on any error
+# Stop execution on any error
 trap "catchFinish" EXIT SIGINT
 
-overrideDefaults() {
-  # CI_CHE_OPERATOR_IMAGE it is che operator image builded in openshift CI job workflow. More info about how works image dependencies in ci:https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
-  export OPERATOR_IMAGE=${CI_CHE_OPERATOR_IMAGE:-"quay.io/eclipse/che-operator:nightly"}
+deployChe() {
+  chectl server:deploy  --telemetry=off --workspace-engine=dev-workspace --platform=openshift --installer=operator --batch
 }
 
-runTests() {
-    # Deploy Eclipse Che applying CR
-    applyOlmCR
-    waitEclipseCheDeployed "nightly"
-    provisionOAuth
-    startNewWorkspace
-    waitWorkspaceStart
-
-    # Dev Workspace controller tests
-    deployDevWorkspaceController
-    waitDevWorkspaceControllerStarted
-
-    sleep 10s
-    createWorkspaceDevWorkspaceController
-    waitWorkspaceStartedDevWorkspaceController
-
-    sleep 10s
-    createWorkspaceDevWorkspaceCheOperator
-    waitWorkspaceStartedDevWorkspaceController
+runTest() {
+  deployChe
+  waitDevWorkspaceControllerStarted
+
+  # wait for 2 min for devworkspace-controller ready to work.
+  sleep 120
+  createWorkspaceDevWorkspaceController
+  waitWorkspaceStartedDevWorkspaceController 
+
+  sleep 120
+  oc get pods -n ${NAMESPACE}
+  oc get routes -n ${NAMESPACE}
+  oc get events -n ${NAMESPACE}
+
+  wsname=$(oc get pods  -n ${NAMESPACE} | grep workspace | awk '{print $1}')
+  oc logs $wsname -c theia-ide -n ${NAMESPACE}
+  oc logs $wsname -c terminal -n ${NAMESPACE}
+
+  # patch pod.yaml 
+  wget https://gist.githubusercontent.com/SkorikSergey/3ce0000301d122b6a41782a98229ccc3/raw/465cbe34cd0bba14e1fd1300f88597b992ac12d2/happy-path-pod.yaml
```

Comment:
You don't need to have a gist with happy path pod and download. You can add it to `.ci/openshift-ci` folder and reuse it in tests